crates/aptos/src/stake/mod.rs

```diff
@@ -297,3 +303,126 @@ impl CliCommand<TransactionSummary> for CreateStakingContract {
             .map(|inner| inner.into())
     }
 }
+
+#[derive(Parser)]
+pub struct UnlockVestingRewards {
```

Comment:
All of these commands should have a doc comment explaining what it does

Reply:
Updated